docs/advanced/document-store/benchmark.md

```diff
@@ -28,11 +28,11 @@ We focus on the following tasks:
 5. **Find by condition**: search existing Documents by `.tags` via {meth}`~docarray.array.mixins.find` in the document store by boolean filters, as described in {ref}`find-documentarray`.
 6. **Find by vector**: retrieve existing Documents by `.embedding` via {meth}`~docarray.array.mixins.find`  using  nearest neighbor search or approximate nearest neighbor search, as described in {ref}`match-documentarray`.
 
-We are interested in the single-query performance on the above tasks, which means the tasks 2,3,4,5,6 are evaluated using one Document at a time, repeatedly, and the average number is reported.
+We are interested in the single query performance on the above tasks, which means tasks 2,3,4,5,6 are evaluated using one Document at a time, repeatedly. We report the average number.
 
 ## Benchmark result
 
-The following table summarizes the result; values are smaller the better (except for `Recall@10`). The best performer of each task is highlighted with bold font:
+The following table summarizes the result. Values are smaller the better (except for `Recall@10`). The best performer of each task is highlighted in **bold**:
```

Comment:
```suggestion
The following table summarizes the result. The smaller the values, the better (except for `Recall@10`). The best performer of each task is highlighted in **bold**:
```
do you think this is more gramatically correct ?

Reply:
Yes